ros/processors/inventory_events_consumer.py

```diff
@@ -0,0 +1,109 @@
+# import threading
+import json
+import logging
+from confluent_kafka import Consumer, KafkaException
+from ros.config import INSIGHTS_KAFKA_ADDRESS, INVENTORY_EGRESS_TOPIC
+from ros.app import app, db
+from ros.models import PerformanceProfile
+
+logging.basicConfig(
+    level='INFO',
+    format='%(asctime)s - %(levelname)s  - %(funcName)s - %(message)s'
+)
+LOG = logging.getLogger(__name__)
+
+
+class InventoryEventsConsumer:
+    """Inventory events consumer."""
+
+    def __init__(self):
+        """Create a Inventory Events Consumer."""
+        self.running = True
+        self.consumer = Consumer({
+            'bootstrap.servers': INSIGHTS_KAFKA_ADDRESS,
+            'group.id': 'inventory-events-consumer',
+            'enable.auto.commit': False
+        })
+
+        # Subscribe to topic
+        self.consumer.subscribe([INVENTORY_EGRESS_TOPIC])
+        self.event_type_map = {
+            'created': self.host_create_update_event,
+            'updated': self.host_create_update_event,
+            'delete': self.host_delete_event,
+        }
+        self.prefix = 'PROCESSING INVENTORY EVENTS'
+
+    def run(self):
+        """Initialize Consumer."""
+        while self.running:
+            msg = self.consumer.poll(1.0)
+            if msg is None:
+                continue
+            if msg.error():
+                print(msg.error())
+                raise KafkaException(msg.error())
+            try:
+                msg = json.loads(msg.value().decode("utf-8"))
+                event_type = msg['type']
+                if event_type in self.event_type_map.keys():
+                    handler = self.event_type_map[event_type]
+                    handler(msg)
+                else:
+                    LOG.info(
+                        'Event Handling is not found for event %s - %s',
+                        event_type, self.prefix
+                    )
+            except json.decoder.JSONDecodeError:
+                LOG.error(
+                    'Unable to decode kafka message: %s - %s',
+                    msg.value(), self.prefix
+                )
+            except Exception as err:
+                LOG.error(
+                    'An error occurred during message processing: %s - %s',
+                    repr(err),
+                    self.prefix
+                )
+            finally:
+                self.consumer.commit()
+
+        self.consumer.close()
+
+    def host_create_update_event(self, _msg):
+        """Process create/update message."""
+        self.prefix = "PROCESSING CREATE/UPDATE EVENT"
+        LOG.info(
```

Comment:
Is the logic for Create/Update to be added here in the same PR?

Reply:
https://github.com/RedHatInsights/ros-backend/pull/23#issue-569559504 
These are the only points I am going to target in this PR.

Reply:
so should we eliminate the `host_create_update_event` part from this PR?

Reply:
Done!